pontoon/base/static/css/style.css

```diff
@@ -1132,7 +1133,7 @@ nav .links li {
 }
 
 .submenu.tabs .links a {
-    padding: 12px 25px;
+    padding: 12px 20px;
```

Comment:
These CSS changes should probably not be a part of this PR?

Reply:
Ooops.